modules/oc-mirror-command-reference-v2.adoc

```diff
@@ -77,4 +77,25 @@ The following tables describe the `oc mirror` subcommands and flags for oc-mirro
 |`--workspace` 
 |Determines string oc-mirror plugin v2 workspace where resources and internal artifacts are generated.
 
+|`--cache-dir string` 
```

Comment:
with the recent changes this could also be specified on the `mirror workflow command lines`, not sure if we need to include that information.

@r4f4  WDYT ?

Reply:
@r4f4 Never mind you are right...